mypy_protobuf/main.py

```diff
@@ -828,6 +829,10 @@ def generate_mypy_grpc_stubs(
 def code_generation() -> Generator[
     Tuple[plugin_pb2.CodeGeneratorRequest, plugin_pb2.CodeGeneratorResponse], None, None
 ]:
+    if len(sys.argv) > 1 and sys.argv[1] in ("-V", "--version"):
+        print(__version__)
```

Comment:
small nitpick, but noteworthy to mention that usually the output for version also prints a program name and not just the version by itself:

```
$ python -V
Python 3.6.8

$ protoc --version
libprotoc 3.14.0

$ pytest -V
pytest 6.2.4

$ black --version
black, version 21.5b1

$ cargo --version
cargo 1.53.0 (4369396ce 2021-04-27)
```

And sometimes, even more information is added, such as copyright, license, etc...

Reply:
good idea.

Something like `mypy-protobuf 2.5` might be helpful